sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample1_HelloWorld.cs

```diff
@@ -9,37 +9,34 @@
 namespace Azure.Data.AppConfiguration.Samples
 {
     [LiveOnly]
+    [NonParallelizable]
     public partial class ConfigurationSamples
     {
         [Test]
-        /*
```

Comment:
Just to confirm: these comments are removed because they're explained in the .md file?

Reply:
Yes. The same thing has been done in KeyVault